src/core/org.freedesktop.systemd1.policy.in

```diff
@@ -70,4 +70,14 @@
                 </defaults>
         </action>
 
+        <action id="org.freedesktop.systemd1.dump">
```

Comment:
this is very confusing. The name suggests this could restrict Dump() access, but it cannot do that.

Reply:
renamed to `bypass-dump-ratelimit`